libsql-server/src/stats.rs

```diff
@@ -56,6 +57,8 @@ pub struct Stats {
     #[serde(skip)]
     namespace: NamespaceName,
 
+    #[serde(default)]
+    id: Uuid,
```

Comment:
```suggestion
    #[serde(default)]
    id: Option<Uuid>,
```

This is how we would do it in rust